pkg/controller/buildrun/generate_taskrun_test.go

```diff
@@ -55,10 +55,10 @@ var _ = Describe("GenerateTaskrun", func() {
 				build, err = ctl.LoadBuildYAML([]byte(test.MinimalBuildahBuild))
 				Expect(err).To(BeNil())
 
-				buildRun, err = ctl.LoadBuildRunYAML([]byte(test.MinimalBuildahBuildRun))
+				buildRun, err = ctl.LoadBRFromBytes([]byte(test.MinimalBuildahBuildRun))
 				Expect(err).To(BeNil())
 
-				buildStrategy, err = ctl.LoadBuildStrategyYAML([]byte(test.MinimalBuildahBuildStrategy))
+				buildStrategy, err = ctl.LoadBSFromBytes([]byte(test.MinimalBuildahBuildStrategy))
```

Comment:
Same here, `BuildStrategy` instead of `BS`.

Reply:
same here. That change is already there. I dont recall how